examples/coordinator-init.yaml

```diff
@@ -0,0 +1,66 @@
+log_level: debug
```

Comment:
it looks like a router config, is it definitely coordinator-init.yaml?

Reply:
it is router-etcd-init.yaml